app/src/main/java/fr/free/nrw/commons/delete/DeleteHelper.java

```diff
@@ -91,12 +92,12 @@ private boolean delete(Media media, String reason) {
                 reason +
                 " ~~~~";
 
-        String logPageString = "\n{{Commons:Deletion requests/" + media.getFilename() +
+        String logPageString = "\n" + "{{Commons:Deletion requests" + media.getFilename() +
```

Comment:
I guess you forgot removing "+" from your previous implementation. This method shouldn't be edited at all if I don't miss any point.